go/pkg/command/command.go

```diff
@@ -45,6 +45,30 @@ func (s InputType) String() string {
 	return fmt.Sprintf("InvalidInputType(%d)", s)
 }
 
+// SymlinkBehaviorType represents how symlinks are handled.
+type SymlinkBehaviorType int
+
+const (
+	// UnspecifiedSymlinkBehavior means following clients.TreeSymlinkOpts
+	// or DefaultTreeSymlinkOpts if clients.TreeSymlinkOpts is null.
+	UnspecifiedSymlinkBehavior SymlinkBehaviorType = iota
+
+	// ResolveSymlink means symlinks are resolved.
+	ResolveSymlink
+
+	// PreserveSymlink means symlinks are kept as-is.
+	PreserveSymlink
+)
+
+var symlinkBehaviorType = [...]string{"UnspecifiedSymlinkBehavior", "ResolveSymlink", "PreserveSymlink"}
+
+func (s SymlinkBehaviorType) String() string {
+	if UnspecifiedSymlinkBehavior <= s && s < SymlinkBehaviorType(len(symlinkBehaviorType)) {
```

Comment:
better to cast `s` to `int`?

Reply:
Hmm, instead of trying to write something different, let me follow how InputType is written.